drivers/hmis/app/graphql/types/hmis_schema/mutation_type.rb

```diff
@@ -41,7 +41,7 @@ class HmisSchema::MutationType < Types::BaseObject
 
     field :create_service_type, mutation: Mutations::CreateServiceType
     field :delete_service_type, mutation: Mutations::DeleteServiceType
-    field :rename_service_type, mutation: Mutations::RenameServiceType
+    field :update_service_type, mutation: Mutations::UpdateServiceType
 
```

Comment:
Does it work to do something like this? We should be in the habit of deprecating instead of removing/renaming as much as possible
```
    field :rename_service_type, mutation: Mutations::UpdateServiceType, deprecation_reason: "Renamed to updateServiceType"
    field :update_service_type, mutation: Mutations::UpdateServiceType
```

Reply:
Yes this works great. Good idea to just use the same UpdateServiceType mutation.